### PR TITLE
feat: Add traceAddress to TransactionEvent

### PR DESCRIPTION
**Pull request by 0xtaf**

### CODE REVIEW

**Review by Vxatz — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "forta-agent-tools",
-  "version": "3.0.0",
+  "version": "3.0.1",
```
Comment (RCantu92) [inline package.json:3]: As was brought up, we could change this to be compatible with Franco's PR.

**Review by RCantu92 — COMMENTED** [review]
One small comment. Outside of that, LGTM

**Review by Vxatz — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "forta-agent-tools",
-  "version": "3.1.0",
+  "version": "3.2.0",
```
Comment (RCantu92) [inline package.json:3]: Instead of `3.2.0`, we can make it `3.1.1`.

**Review by RCantu92 — COMMENTED** [review]
Suggestion for a different version number.

File: package.json (hunk 2 above)
Comment (0xtaf) [reply package.json]: done 👍

**Review by RCantu92 — APPROVED** [review]
LGTM 👍

**Pull request merged**